app/views/devise/registrations/new.html.erb

```diff
@@ -6,11 +6,11 @@
   <%= devise_error_messages! %>
 
   <div class="form-group">
-    <%= f.text_field :login, class: 'form-control', placeholder: t(".username"), autofocus: true %>
+    <%= f.text_field :login, class: 'form-control', placeholder: true, autofocus: true %>
```

Comment:
The publify platform needs to be useable by an international audience of non-english speakers. Please do not remove placeholder text!